platform/ext/target/nordic_nrf/nrf5340/services/src/tfm_platform_system.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019, Arm Limited. All rights reserved.
```

Comment:
Perhaps you should add ` * Copyright (c) 2020, Nordic Semiconductor ASA. All rights reserved.`.

Reply:
Hm, I don't want to do this yet - this file is almost identical to most other equivalent headers, so I'd keep ARM's copyright for now. (This will change if we add significant additional implementation here)